src/python/pants/backend/python/target_types.py

```diff
@@ -548,6 +548,25 @@ class PexIncludeToolsField(BoolField):
     )
 
 
+class RunInlineField(BoolField):
+    alias = "run_inline"
```

Comment:
Alternatives:
- Call it `run_in_repo` (or similar) to capture intent. (Instead of introducing a new word: "inline")
- Flip it around. I expect most users assume `run` runs the code in-repo, so make this field align with expectations. `run_in_sandbox` defaulting to `True`. Then the flag becomes `--no-run-in-sandbox`.

Reply:
Ah yes. Good point. Might be worth pontificating a flag akin to `skip` being a flag and a field.

Reply:
> Note that this concept of running "inline" is Python-specific, not generic to the run goal.

This is specific to any run-as-is (non-compiled, uncompiled?) language. So languages like Ruby, JS, or Shell would all be valid.